diff --git a/content/public/browser/web_contents_receiver_set.h b/content/public/browser/web_contents_receiver_set.h
index 36ba25ee87ea9d120bf356263202eb9b4ee9db7f..9143d79941887915cf5a595bb60b23926afe8454 100644
--- a/content/public/browser/web_contents_receiver_set.h
+++ b/content/public/browser/web_contents_receiver_set.h
@@ -28,6 +28,11 @@ namespace subresource_redirect {
 class SubresourceRedirectObserver;
 }
 
+class BraveDrmTabHelper;
+namespace brave_shields {
+class BraveShieldsWebContentsObserver;
+}
+
 namespace content {
 
 class RenderFrameHost;
@@ -96,6 +101,8 @@ class WebContentsFrameReceiverSetPassKey {
   friend class ::printing::PrintManager;
   friend class ::subresource_redirect::SubresourceRedirectObserver;
   friend class TestFrameInterfaceBinder;
+  friend class ::BraveDrmTabHelper;
+  friend class ::brave_shields::BraveShieldsWebContentsObserver;
   FRIEND_TEST_ALL_PREFIXES(WebContentsReceiverSetBrowserTest,
                            OverrideForTesting);
 };
